rpc/core/health.go

```diff
@@ -31,6 +32,6 @@ import (
 // 	"jsonrpc": "2.0"
 // }
 // ```
-func Health() (*ctypes.ResultHealth, error) {
+func Health(ctx *rpctypes.Context) (*ctypes.ResultHealth, error) {
```

Comment:
ctx n ot used. Also, the code in the comment does not use the ctx: [`result, err := client.Health()` ](https://github.com/tendermint/tendermint/pull/3269/files#diff-8f61cabd9713bfb1c674667f8dbdc189L21)

Reply:
Comment is correct. RPC client has a different API than our core in-house RPC API.